plugin/PDFContextMenu.qml

```diff
@@ -0,0 +1,135 @@
+/****************************************************************************************
+  **
+  ** Copyright (C) 2013-2016 Jolla Ltd., Damien Caliste
+  ** Contact: Martin Jones <martin.jones@jollamobile.com>
+  ** Contact: Damien Caliste <dcaliste@free.fr>
+  ** All rights reserved.
+  ** 
+  ** You may use this file under the terms of BSD license as follows:
```

Comment:
Well, in fact, I copied the code from Silica/ContextMenu.qml (and keep the license from that file) not from the browser, so there may not be any relicensing issue because Jolla is only copyright owner ? Right ? You can see it because function names are those from the Silica ContextMenu item and not exactly the one of the BrowserContextMenu file.

Beside, I can rewrite the two copied functions, but it seems to me that it was more in the spirit of the license to copy the code and keep the license than to rewrite the same idea with different variable names and not mention the original copyright owner of the implementation.